examples/complete/main.tf

```diff
@@ -127,6 +126,12 @@ module "secrets_manager_disabled" {
 # Supporting Resources
 ################################################################################
 
+ephemeral "random_password" "password" {
```

Comment:
ephemeral outside of the module is fine, but I think we should have ephemeral within the module as well. This way, users can just call the module without any external resources/ephemerals.

Reply:
It looks like OpenTofu doesn't support ephemeral variables yet, so if we add ephemeral variables into the module, it will break it for OpenTofu users.

https://opentofu.org/docs/language/values/variables/#arguments

Reply:
re-added using ephemeral resource https://github.com/terraform-aws-modules/terraform-aws-secrets-manager/commit/9682e45340bb05af8ddcff6abaf077219e45574b